docs/cn/Configuring-Alluxio-with-Minio.md

```diff
@@ -39,7 +39,17 @@ Minio是为云应用程序和DevOps构建的对象存储服务器。 Minio提供
 
 在`conf/alluxio-site.properties`文件中要修改的所有字段如下所示：
 
-{% include Configuring-Alluxio-with-Minio/minio.md %}
+```properties
+alluxio.underfs.address=s3a://<MINIO_BUCKET>/<MINIO_DIRECTORY>
+alluxio.underfs.s3.endpoint=http://<MINIO_ENDPOINT>/
+alluxio.underfs.s3.disable.dns.buckets=true
+alluxio.underfs.s3a.inherit_acl=false
+alluxio.underfs.s3.proxy.https.only=<USE_HTTPS>
```

Comment:
Do you have to remove these few https related properties?

Reply:
Did not remove it for Chinese docs as the text still refers the properties. I was expecting that someone will fix the Chinese docs later.